cmd/funnel/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+
+	"github.com/eagraf/habitat-new/internal/node/config"
+	"github.com/eagraf/habitat-new/internal/node/logging"
+	"tailscale.com/tsnet"
+)
+
+func main() {
+	// read command line args
+	port := os.Args[1]
+	hostName := os.Args[2]
+
+	logger := logging.NewLogger()
+
+	nodeConfig, err := config.NewNodeConfig()
+	if err != nil {
+		logger.Fatal().Err(err).Msg("error loading node config")
+	}
```

Comment:
instead of using node config can we take in TailscaleAuthkey as a cli flag or argument?

i think node config is overloaded right now so trying to remove stuff from it. this should let us remove authkey

Reply:
TailscaleAuthkey is usually an env variable so I wanted to reuse that. Agree that config is overloaded and hard to use but will leave as in until we have a better solution